.github/scripts/check-missing-readme-generation.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
```

Comment:
Would be great to know where this runs too: 
```suggestion
echo "Running script relative to `pwd`"
```

Not running it in the repo root might result in missing docify.md files when `find` is called.